Documentation/README_Solver.md

```diff
@@ -35,16 +35,16 @@ In order to use the Solver system, simply add one of the components listed above
 
 The *Tracked Target Type* property of the [`SolverHandler`](xref:Microsoft.MixedReality.Toolkit.Utilities.Solvers.SolverHandler) component defines the point of reference all solvers will use to calculate their algorithms. For example, a value type of [`Head`](xref:Microsoft.MixedReality.Toolkit.Utilities.TrackedObjectType.Head) with a simple [`SurfaceMagnetism`](xref:Microsoft.MixedReality.Toolkit.Utilities.Solvers.SurfaceMagnetism) component will result in a raycast from the head and in the direction of the user's gaze for solving what surface is hit. Potential values for the `TrackedTargetType` property are:
 
-* *Head* : Point of reference from the camera
-* *MotionController*: Point of reference from a motion controller
+* *Head* : Point of reference is the transform of the main camera
+* *ControllerRay*: Point of reference is the line pointer on a controller (i.e motion controller or hands)
```

Comment:
Nit: I'm not quite sure if this is the beginning or end of a "line pointer" or what a "line pointer" is on a hand?

Reply:
```suggestion
* *ControllerRay*: Point of reference is the [`LinePointer`](xref:Microsoft.MixedReality.Toolkit.Input.LinePointer) transform on a controller (i.e pointer origin on a motion controller or hand controller) pointing in the direction of the line ray
```